Document "nodes" in navigation-parameters-reference.md (#3468)
SAP/luigi: docs/navigation-parameters-reference.md

- **type**: function
- **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.

### nodes

> - **type**: array

Author: Yes, you're right. I'm missing some technical details. Is it supposed to be an `Object`? Maybe @JohannesDoberer knows more. Also, the behavior we tested on Fiddle - someone deletes the root node (?) and what should be the result? Maybe we can add a note about it.

> - **type**: array

Author: Update: changes were made in the latest commit.

- **type**: function
- **description**: allows you to invoke and execute a specific function on the global level when a request to navigate to the node occurs. The function receives two node objects as input parameters: the previous node and current node, as described in the configuration.

### nodes
- **type**: array | object

> - **description**: You can define navigation nodes using the `nodes:` attribute. First-level node are also referred to as "root nodes". Second-level notes can be defined inside a [children](#children) array below the root node. 

Reviewer: ```suggestion
- **description**: You can define navigation nodes using the `nodes:` attribute. First-level nodes are also referred to as "root nodes". Second-level nodes can be defined inside a [children](#children) array below the root node. 
```

> Instead of an array, `nodes:` can also be defined as a single node object which then serves as the root node. 

Reviewer: ```suggestion
In addition to an array, `nodes:` can also be defined as a single node object which then serves as the root node. 
```
